packages/api/src/utils/index.ts

```diff
@@ -85,24 +86,20 @@ export function normalizeConfig (
   const chains: Array<{ networks: NetworkConfigMap }> = Object.values(
     config.chains
   )
-
-  // Extracts networks with its key label from chains object
-  const networksConfigMap: Array<NetworkConfigMap> = chains.reduce(
-    (acc: Array<NetworkConfigMap>, network) => {
-      return [...acc, network.networks]
-    },
-    []
-  )
   // Network Config list deleting key label
-  const configs: Array<FeedConfig> = networksConfigMap.flatMap(
-    networkConfigMap =>
-      Object.values(networkConfigMap).map((feed, index) => ({
-        ...feed,
-        chain: Object.keys(networkConfigMap)[index]
-      }))
-  )
+  const networksConfigMap = chains.flatMap((network: Chain) => {
+    const networks = Object.values(network.networks).map(
+      (feedConfig: FeedConfig, index) => ({
+        ...feedConfig,
+        chain: network.name,
+        network: Object.keys(network.networks)[index]
+      })
+    )
+    return [...networks]
```

Comment:
Why do we need to destructure it?
```suggestion
    return Object.values(network.networks).map(
      (feedConfig: FeedConfig, index) => ({
        ...feedConfig,
        chain: network.name,
        network: Object.keys(network.networks)[index]
      })
    )
```